libs/designsystem/helpers/src/content-mutation-observer.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { Component, ElementRef, Input, OnDestroy } from '@angular/core';
+import { createComponentFactory, Spectator } from '@ngneat/spectator';
+import { TestHelper } from '@kirbydesign/designsystem/testing';
+import { observeContent } from './content-mutation-observer';
+
+@Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
+  selector: 'content-observer-host',
+  template: '<div [innerHtml]="content"></div>',
+})
+class ContentObserverHostComponent implements OnDestroy {
+  @Input() content: string = 'Initial content';
+
+  private mutationObserverDisconnectFn: () => void;
```

Comment:
Consider naming - IMO it's an implementation detail what the return function actually does (unobserve, disconnect or something else)